tests/test_demog.yaml

```diff
@@ -0,0 +1,24 @@
+description: Two deme model with migration and size changes.
```

Comment:
It'd be very cool if demes made its own examples available as package data.

Reply:
I agree - should open an issue for that over in the demes repo

Reply:
There are some examples perhaps, now that I think of it. I can see if there are and try to use some of those instead of having the example yaml file here.

Reply:
Yeah, I'm just not sure they go in via "pip install".